Source/Prism/Common/ParametersExtensions.cs

```diff
@@ -46,6 +48,8 @@ public static bool TryGetValue<T>(this IEnumerable<KeyValuePair<string, object>>
                         value = (T)kvp.Value;
                     else if (typeof(T).IsAssignableFrom(kvp.Value.GetType()))
                         value = (T)kvp.Value;
+                    else if (type.IsEnum && Enum.TryParse<T>(kvp.Value, true, out var enumValue))
```

Comment:
this should be typeof(T).IsEnum... as is this will not compile

Reply:
done